package.json

```diff
@@ -123,7 +127,7 @@
     ]
   },
   "engines": {
-    "node": ">=14.21.0 <17 || >=18 <19"
```

Comment:
Is this change required for this PR?

Reply:
Technically we do support node 19+ right now, wanted to get rid of these warnings

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'parse@4.0.0',
npm WARN EBADENGINE   required: { node: '>=14.21.0 <17 || >=18 <19' },
npm WARN EBADENGINE   current: { node: 'v19.3.0', npm: '9.2.0' }
npm WARN EBADENGINE }
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'parse-server@6.0.0-alpha.30',
npm WARN EBADENGINE   required: { node: '>=14.21.0 <17 || >=18 <19' },
npm WARN EBADENGINE   current: { node: 'v19.3.0', npm: '9.2.0' }
npm WARN EBADENGINE }
```

Reply:
Got it, we need to do this in a separate PR please. This merits a distinct changelog entry, it's a bug fix I'd say since we forgot to change this from <19 to <20 (we need to keep an upper range) and I think there is even a strict setting in npm where the installer will abort if there is such a conflict.

Reply:
Opened https://github.com/parse-community/Parse-SDK-JS/issues/1731 to track this